app/frontend/src/features/messages/messagelist/MomentIndication.tsx

```diff
@@ -0,0 +1,15 @@
+import { Box } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import React from "react";
+import moment from "moment";
+
+const useStyles = makeStyles({ root: {} });
+
+export interface MomentIndicationProps {
+  date: Date;
+}
+
+export default function TimeInterval({ date }: MomentIndicationProps) {
+  const classes = useStyles();
+  return <Box className={classes.root}>{moment(date).fromNow()}</Box>;
```

Comment:
Yes, moment can be very big. Especially with all locales. I think we should use it now(maybe behind a facade) and replace if needed. I also think the size of moment can be minimized somewhat. But for now let's go with it.

Reply:
I also think that the size of moment will not cause problems for the type of website we are building. It would be a different story if we had to convince people with small bandwidths to buy something on their first visit.